apps/theme/templates/greencheck/partials/directory_results.html

```diff
@@ -0,0 +1,114 @@
+
+		<section class="country-list my-2 lg:my-4">
+
+			<h2 class="text-2xl my-4 border-b-2 border-black">{{ country.grouper }} ({{ country.list|length }})</h2>
+			
+			{% for obj in country.list %}
+				<article class="provider-listing lg:w-3/4 my-6 lg:mr-0 lg:ml-auto py-6 last-of-type:pb-2 border-b-2 border-neutral-200 last:border-0">
+					<div class="prose mx-auto lg:ml-0 lg:mr-auto">
+
+						<h4 class="inline-block mb-0 font-bold text-xl" id="{{ obj.id }}">
+							{% if obj.website %}						
+								<a class="no-underline" target="_blank" rel="noopener noreferrer" href="https://{{ obj.website|urlencode }}">	
+							{% endif %}
+
+							{{ obj.name }}
+
+							{% if obj.website %}
+								<img class="utility-icon external-link-icon" width="16px" height="16px" src="{% static 'img/icon-external-link.svg' %}" alt="Visit {{ obj.name }}'s website"/>
+								</a>
+							{% endif %}
+						</h4>
+
+						{% if obj.description %}
+							<p class="text-neutral-700 mt-2 mb-3">
+								{{ obj.description }}
+							</p>
+						{% endif %}
+
+						{% if obj.services.names %}
+							<ul class="my-4 pl-0">
+								{% for service in obj.services.all %}
+									<li class="service-label inline-block">{{ service.name }}</li>
+								{% endfor %}
+
+								{% comment %}
+									Every host in our directory used to at least offer shared hosting.
+									Below is a placeholder to see how it would look with live data.
+									{% else %}
+									<li class="service-label inline-block">Compute: Shared Hosting for Websites</li>
+								{% endcomment %}
+							</ul>
+						{% endif %}
+						
+						{% if obj.public_supporting_evidence %}
+							<details class="mb-4">
+								<summary>
+									<span class="">Published supporting evidence</span>
+								</summary>
+
+								<ul class="mt-3">
+									{% for evidence in obj.public_supporting_evidence %}
+										<li class="m-3">
+											<a class="my-2" target="_blank" rel="noopener noreferrer" href="{{ evidence.link }}">{{ evidence.title }}</a>
+
+											{% if evidence.description %}
+												<div class="evidence-description text-sm">
+													{{ evidence.description|linebreaks }}
+												</div>
+											{% endif %}
+										</li>
+									{% endfor %}
+								</ul>
+							</details>
+						{% endif %}
+
+					</div>
+
+					{% if not obj.public_supporting_evidence and not obj.description %}
+						<details class="">
+							<summary class="text-right text-sm text-neutral-400">Help improve this listing</summary>
+
+							<p class="max-w-xs md:max-w-md text-sm text-justify ml-auto mr-0 mt-2 text-neutral-400">
+							Once a hosting provider gets verified with us, we ask them to keep
+							their listing up to date and refresh their evidence at least annually.</p>
+
+							<p class="max-w-xs md:max-w-md text-sm text-justify ml-auto mr-0 mt-2 text-neutral-400">
+							<span class="text-neutral-500">Are you a customer of this provider?</span> We find
```

Comment:
We have a sample emails page, linked below - would you mind linking to that? It might be a separate card to add a sample email to ask for this change, but having at least some copy and pastable content on a url we can send people to would provide an action we can see if people are taking.

https://www.thegreenwebfoundation.org/sample-emails/

Reply:
Yes happy to add that in. I will draft a sample email on that page at the same time to cover this case so we have at least something that can be improved later.

Reply:
Sample email made on the page, and the other emails and content in general given a bit of love and attention. I think that closes this card off for now.